homeworks/src/andriiarzhanov/com/hackerrank/Day4_Class_vs_Instance/Day4_Class_vs_Instance.java

```diff
@@ -0,0 +1,26 @@
+package andriiarzhanov.com.hackerrank.Day4_Class_vs_Instance;
+
+import java.util.Scanner;
+
+/**
+ * Created by arzhanov on 19.04.17.
+ */
+public class Day4_Class_vs_Instance {
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int T = sc.nextInt();
```

Comment:
This part is default code from task. I copied all code from task just for correct work of project
![image](https://cloud.githubusercontent.com/assets/2251075/25305337/2493dd96-2782-11e7-987b-97bff03ec1a1.png)